msprime/provenance.py

```diff
@@ -20,6 +20,13 @@
 Common provenance methods used to determine the state and versions
 of various dependencies and the OS.
 """
+import base64
+import importlib
+import json
+import marshal
+import logging
+import types
```

Comment:
Minor style point here. I like to split imports up into three groups: standard library, external imports and local imports. So, here it would be 
```python
import base64
￼import importlib
￼import json
￼import marshal
￼import logging
￼import types

import numpy
￼import tskit

import _msprime
```
This is a useful convention I think, as it lets you see in a glance what the module depends on.

Reply:
Cool, we should use https://github.com/timothycrosley/isort to do this as part of https://github.com/tskit-dev/tskit/issues/470 (which I'm assuming once we're happy with we'll bring across to the other repos?)

Reply:
Great idea! Yes, I'm imagining us getting the tskit repo fully up to date with new linting stuff before porting across to msprime, kastore, tsinfer etc.